test/conftest.py

```diff
@@ -23,180 +23,182 @@
 
 
 logger = logging.getLogger(__name__)
-logging.getLogger('boto').setLevel(logging.INFO)
-logging.getLogger('botocore').setLevel(logging.INFO)
-logging.getLogger('factory.py').setLevel(logging.INFO)
-logging.getLogger('auth.py').setLevel(logging.INFO)
-logging.getLogger('connectionpool.py').setLevel(logging.INFO)
+logging.getLogger("boto").setLevel(logging.INFO)
+logging.getLogger("botocore").setLevel(logging.INFO)
+logging.getLogger("factory.py").setLevel(logging.INFO)
+logging.getLogger("auth.py").setLevel(logging.INFO)
+logging.getLogger("connectionpool.py").setLevel(logging.INFO)
 
 DIR_PATH = os.path.dirname(os.path.realpath(__file__))
 
 # these regions have some p2 and p3 instances, but not enough for automated testing
 NO_P2_REGIONS = [
-    'ca-central-1',
-    'eu-central-1',
-    'eu-west-2',
-    'us-west-1',
-    'eu-west-3',
-    'eu-north-1',
-    'sa-east-1',
-    'ap-east-1',
-    'me-south-1'
+    "ca-central-1",
+    "eu-central-1",
+    "eu-west-2",
+    "us-west-1",
+    "eu-west-3",
+    "eu-north-1",
+    "sa-east-1",
+    "ap-east-1",
+    "me-south-1",
 ]
 NO_P3_REGIONS = [
-    'ap-southeast-1',
-    'ap-southeast-2',
-    'ap-south-1',
-    'ca-central-1',
-    'eu-central-1',
-    'eu-west-2',
-    'us-west-1'
-    'eu-west-3',
-    'eu-north-1',
-    'sa-east-1',
-    'ap-east-1',
-    'me-south-1'
+    "ap-southeast-1",
+    "ap-southeast-2",
+    "ap-south-1",
+    "ca-central-1",
+    "eu-central-1",
+    "eu-west-2",
+    "us-west-1" "eu-west-3",
```

Comment:
Missing `,` here. This will cause the two strings to be concatenated. Is this intentional?

Reply:
This is not intentional. Will submit a separate PR to fix this and add recently added regions to the list.